failpoint.go

```diff
@@ -82,21 +82,18 @@ func (fp *Failpoint) EnableWith(inTerms string, action func() error) error {
 }
 
 // Disable stops a failpoint
-func (fp *Failpoint) Disable() error {
+func (fp *Failpoint) Disable() {
 	select {
 	case <-fp.waitChan:
-		return ErrDisabled
```

Comment:
Seems the `ErrDisabled` is useless.

Reply:
The constant is still used at `failpoint.Eval` and `failpoint.Status` to report error if a failpoint is disabled so that it cannot eval or show status.